addon/services/-sustains.js

```diff
@@ -1,60 +0,0 @@
-import Ember from 'ember';
-import SustainModel from '../lib/sustain';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
This is the only time we use `ember-getowner-polyfill`. Safe to remove from package.json now.